apps/marketplace/src/components/MoreSongs.tsx

```diff
@@ -0,0 +1,29 @@
+import { FunctionComponent } from "react";
+import { Box, useTheme } from "@mui/material";
+import Songs from "./Songs";
+import { mockArtist, mockSongs } from "../temp/data";
+
+const MoreSongs: FunctionComponent = () => {
+  const theme = useTheme();
+
+  const artist = mockArtist;
+  const songs = mockSongs;
+  const artistFullName = `${artist.firstName} ${artist.lastName}`;
+
+  const title = (
+    <Box component="span">
+      MORE FROM&nbsp;
+      <Box component="span" sx={ { color: theme.colors.music } }>
+        { artist.stageName || artistFullName }
+      </Box>
```

Comment:
The title doesn't seem to properly center on a mobile view like the "Similar Songs" title.